README.md

```diff
@@ -1,14 +1,114 @@
+with `CONDUIT_TRACKING_ID` ordering column.
+
+The Ack method collects the `CONDUIT_TRACKING_ID` of those records that have been successfully applied, in order to
+remove them later in a batch (every 5 seconds or when the connector is closed).
+
+### Position
+
+Example of the position:
+
+```
+{
+  "mode": "snapshot",
+  "last_processed_val": 1
+}
+```
+
+`mode` is a mode of an iterator (`snapshot` and `cdc`);
+
+`last_processed_val` is a last processed element value of an ordering column.
+
+### Configuration Options
+
+| name             | description                                                                         | required |
+|------------------|-------------------------------------------------------------------------------------|----------|
+| `url`            | String line for connection to DB2.                                                  | **true** |
+| `table`          | The name of a table in the database that the connector should write to, by default. | **true** |
+| `keyColumn`      | Column name records should use for their `Key` fields.                              | **true** |
+| `orderingColumn` | Column name of a column that the connector will use for ordering rows.              | **true** |
+| `columns`        | List of column names that should be included in each Record's payload.              | false    |
+| `batchSize`      | Size of rows batch. Min is 1 and max is 100000.                                     | false    |
+
+## Destination
+
+The Oracle Destination takes a `record.Record` and parses it into a valid SQL query. The Destination is designed to
+handle different payloads and keys. Because of this, each record is individually parses and upsertes.
+
+### Table name
+
+If a record contains a `table` property in its metadata it will be inserted in that table, otherwise it will fall back
+to use the table configured in the connector. This way the Destination can support multiple tables in the same
+connector, provided the user has proper access to those tables.
+
+### Upsert Behavior
+
+If the target table already contains a record with the same key, the Destination will upsert with its current received
+values. Because Keys must be unique, this can overwrite and thus potentially lose data, so keys should be assigned
+correctly from the Source.
+
+### Configuration Options
+
+| name        | description                                                                         | required |
+|-------------|-------------------------------------------------------------------------------------|----------|
+| `url`       | String line for connection to DB2.                                                  | **true** |
+| `table`     | The name of a table in the database that the connector should write to, by default. | **true** |
+| `keyColumn` | Column name uses to detect if the target table already contains the record.         | false    |
```

Comment:
Could you please remind me why key column is not required?

Reply:
I've made a mistake. This field is required.